lib/src/spec.rs

```diff
@@ -94,6 +96,18 @@ pub struct BootEntry {
     pub ostree: Option<BootEntryOstree>,
 }
 
+#[derive(Debug, Clone, Serialize, Deserialize, PartialEq, Eq, JsonSchema)]
+#[serde(rename_all = "camelCase")]
+#[non_exhaustive]
+/// The detected type of running system.  Note that this is not exhaustive
+/// and new variants may be added in the future.
+pub enum HostType {
+    /// The current system is deployed in a bootc compatible way.
+    BootcHost,
+    /// The current system is running as a container that is bootable.
+    BootcContainer,
```

Comment:
The `cC` here "stutters" a bit i.e. it's "containerContainer" but I am not sure of a better term.

Reply:
Hmmmm....yes, I think it is more elegant, but also more work to parse for the default/simple use cases.  Unsure.

Reply:
Maybe what's really simpler is to drop the container detection out entirely.  It's not totally clear to me that it has a use case.  Yeah...let's do that.